pkg/controller/user-cluster-controller-manager/ipam/ipam_controller_test.go

```diff
@@ -59,7 +59,7 @@ func TestSingleCIDRAllocation(t *testing.T) {
 	}
 
 	resultMachine := &clusterv1alpha1.Machine{}
-	if err := r.Get(context.Background(), types.NamespacedName{Name: "Malcolm"}, resultMachine); err != nil {
+	if err := r.Get(context.Background(), types.NamespacedName{Name: m.Name, Namespace: m.Namespace}, resultMachine); err != nil {
```

Comment:
Machines are namespaced and the new fakeclient does not find them if we do not specify the namespace.